README.md

```diff
@@ -76,21 +107,29 @@ for a link to the latest version of `ax_prog_cc_for_build.m4`.
     $ ./configure --prefix=/home/luser
 ```
 
-   If your stdll headers and libraries are not under any standard path, you will
-   need to pass the paths to your files to the configure script. For instance:
+If your stdll headers and libraries are not under any standard path, you will
+need to pass the paths to your files to the configure script.
+**Note:** When compiling on AIX, `CFLAGS` and `LDFLAGS` must be set to the
+correct paths where it can find openldap libraries and header files correctly.
 
-```
+If using the `openldap-devel` package from the [AIX Toolbox](https://www.ibm.com/support/pages/aix-toolbox-open-source-software-downloads-alpha#O),
+then `CFLAGS` and `LDFLAGS` must be set to `-I/opt/freeware/include` and
+`-L/opt/freeware/lib`, respectively, before or with the `./configure`
+invocation. For instance,
+
+```bash
     $ CPPFLAGS="-L/path/lib" LDFLAGS="-I/path/include" ./configure
 ```
 
-   See `./configure --help` for info on various options. The default behavior is
-   to build a default token implicitly. For the s390 platform, the default token
-   is ICA. For other platforms, the default token is the software token. Other
-   tokens may be enabled using the corresponding `--enable-<tok>` configuration
-   option provided the appropriate libraries are available.
+See `./configure --help` for info on various options. The default behavior is
+to build a default token implicitly. For the s390 platform, the default token
+is ICA. For AIX, it is CCA and software tokens. For other platforms, the default
```

Comment:
Well, the description about the default tokens is not entirely correct (not your fault, was incorrect since long time already). I actually would not talk about a "default token" at all. On Linux, it will basically enable all tokens where the build prereqs are fulfilled. This is true for s390x as well as other Linux architectures. Certainly, s390 specific tokens such as the ICA token and the EP11 token will only be built on s390x, simply because the libica library can only be available on s390x, thus the build prereq for them can only be fulfilled on s390x.

So maybe we should rephrase this a bit, while we re at it anyway.